templates/upload/index.html.twig

```diff
@@ -0,0 +1,34 @@
+{% extends 'base.html.twig' %}
+
+{% block body %}
+
+    <p>
+        {% if persons_count is defined %}
```

Comment:
too many if, you have an object result , you just need to check if this object is null or not and then render data or not